docs/en/sql-reference/statements/select/index.md

```diff
@@ -141,6 +141,269 @@ You can use synonyms (`AS` aliases) in any part of a query.
 
 The `GROUP BY` and `ORDER BY` clauses do not support positional arguments. This contradicts MySQL, but conforms to standard SQL. For example, `GROUP BY 1, 2` will be interpreted as grouping by constants (i.e. aggregation of all rows into one).
 
+## Select modifiers {select-modifiers}
+
+You can modify the results returned from a `SELECT` query, as follows.
+
+### Apply {select-apply}
+
+The `APPLY` function allows you to use aggregate functions on multiple columns. 
```

Comment:
It is work not only for aggregate functions, but another too. Must be something this: "The `APPLY` operator allows you to invoke a table-valued function for each columns returned by an outer table expression of a query."